### PR TITLE
Markdown library update, table support, a new post

### PR DESCRIPTION
**Pull request by ForNeVeR**

### CODE REVIEW

File: ForneverMind/posts/en/2021-04-10.net-support-dates.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+    title: .NET Support Dates
+    description: This post describes the current .NET support policy and dates which are sometimes hard to figure out from the official documentation.
+---
+
+Once in a while, I want to know which versions of .NET are currently officially supported. Unfortunately, itr may be hard to figure this out from the official Microsoft documentation. So I've decided to collect this information in this post. I intend to update this information when new .NET versions come out in the future, and with more precise dates if they're available.
```
Comment (Minoru) [inline ForneverMind/posts/en/2021-04-10.net-support-dates.md:5]: s/itr/it/

File: ForneverMind/posts/en/2021-04-10.net-support-dates.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+| 4.7.1                  | Yes                    | 2022-01-11          | 2027-01-12              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2016 Lifecycle][lifecycle.windows-server-2016] |
+| 4.7.2                  | Yes                    | 2024-01-09          | 2029-01-09              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2019 Lifecycle][lifecycle.windows-server-2019] |
+| 4.8                    | Yes                    | 2024-01-09          | 2029-01-09              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2019 Lifecycle][lifecycle.windows-server-2019] |
+
+### Notes
+
+I wasn't able to find any sources for actual .NET Framework 3.5 support dates. [Lifecycle FAQ][docs.lifecycle-faq] sends the reader to the [Lifecycle Policy][lifecycle.net-framework] document, which hasn't .NET Framework 3.5 listed as a separate entry. The [clarification on the support life cycle][support.lifecycle-clarification] lists it as unsupported, though it has no dates:
```
Comment (Minoru) [inline ForneverMind/posts/en/2021-04-10.net-support-dates.md:36]: "which hasn't .NET listed" doesn't sound like correct English to me. Perhaps "doesn't have .NET listed" is better?

**Review by Minoru — COMMENTED** [review]
Hey, I found a typo and a potential mistake, please take a look!

**Comment by ForNeVeR** [comment]
Thanks! Fixed in #110.

**Pull request merged**